docs/content/reference/static-configuration/file.yaml

```diff
@@ -421,3 +425,5 @@ experimental:
   devPlugin:
     goPath: foobar
     moduleName: foobar
+  http3: true
+  kubernetesGateway: true
```

Comment:
This was added by `go generate` but is unrelated to this PR.

Reply:
I think it's an artifact of your rebase.